manifests/c/Cryptomator/Cryptomator/1.7.5/Cryptomator.Cryptomator.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Cryptomator.Cryptomator
+PackageVersion: 1.7.5

```

```suggestion
PackageVersion: 1.7.5.4400
```
This should match control panel. The version folder name will need to be updated when this field is changed.

---

Done.